libs/transactions/helpers/units.ts

```diff
@@ -0,0 +1,105 @@
+import { toBN } from "web3-utils";
+import type BN from "bn.js";
+
+/**
+ * Validates that given value is string
+ *
+ * @function isString
+ * @param {unknown} value transaction value
+ * @returns {boolean}
+ */
+export function isString(s: unknown): boolean {
+  return typeof s === "string" || s instanceof String;
+}
+
+/**
+ * Validates that given value string does not start with `-` ie negative
+ *
+ * @function isPositive
+ * @param {string} value transaction value
+ * @returns {boolean}
+ */
+export function isPositive(value: string): boolean {
+  return value.substring(0, 1) !== "-";
+}
+
+/**
+ * Validates that given value is not just `.`
+ *
+ * @function isValueValid
+ * @param {string} value transaction value
+ * @returns {boolean}
+ */
+export function isValueValid(value: string): boolean {
+  return value !== ".";
```

Comment:
What kind of value are we validating? maybe we could be more explicit about what kind of value we are dealing with and maybe using a library if it is about numbers. In this case the following values are valid `undefined`, `null`, `""`, etc.

Reply:
TBD validations && toBaseUnit